build.gradle

```diff
@@ -23,26 +23,25 @@ compileJava {
 minecraft {
     version = "${project.minecraft_version}-${project.forge_version}"
     runDir = "run"
-    
+
     // the mappings can be changed at any time, and must be in the following format.
     // snapshot_YYYYMMDD   snapshot are built nightly.
     // stable_#            stables are built at the discretion of the MCP team.
     // Use non-default mappings at your own risk. they may not always work.
     // simply re-run your setup task after changing the mappings to update your workspace.
-    mappings = project.forge_mappings
```

Comment:
Why did you make this a hard mapping instead of simply changing the value in gradle.properties?

Reply:
I've already fixed this on master, so we will make sure that we use the changes on master for this file.